website/versioned_docs/version-1.0/fb/fbcode.md

```diff
@@ -22,8 +22,10 @@ Hydra includes buck TARGETS you can use in fbcode. In general, if there is TARGE
 1. You can depend on the TARGETS to use Hydra or a plugin.
 2. The TARGETS contains a runnable example.
 
-targets are under `github/facebookresearch/hydra_1.0`:
-- `:hydra` : Primary target to use in most cases. Includes `hydra_oss` and the `fbcode_defaults`.
+An example TARGET file can be accessed [here](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/examples/tutorials/basic/your_first_hydra_app/6_composition).
+
+targets are under `github/facebookresearch/hydra`:
+- [:`hydra`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra) : Primary target to use in most cases. Includes `hydra_oss` and the `fbcode_defaults`.
 - `:hydra_oss` : Vanilla Hydra without any Facebook specific targets.
-- `plugins/...`: Plugins that has TARGETS file are runnable in fbcode.
-- `examples/...`: Examples that has a TARGETS file are runnable in `fbcode`.
+- [`plugins`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/plugins): Plugins that have TARGETS file are runnable in fbcode.
+- [`examples`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/examples): Examples that have a TARGETS file are runnable in `fbcode`.
```

Comment:
Can you inline an example TARGETS (with a link)? 5_composition can be a gooe one.